server/tailwind.config.js

```diff
@@ -55,6 +55,9 @@ module.exports = {
           light: '#ffefc9',
           DEFAULT: '#6b4b00',
         },
+        'civiform-teal': {
```

Comment:
praise:  Great idea to add this to what's currently serving as our design tokens.